src/components/Book.js

```diff
@@ -1,21 +1,41 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 function Book({ book, removeBook }) {
   return (
     <div>
-      <div className="book-section" key={book.item_id}>
+      <div className="book-section" key={book.itemId}>
         <div className="book-innerSection">
           <span className="book-category">{book.category}</span>
-          <p className="book-title">{book.title}</p>
+          <h5 className="book-title">{book.title}</h5>
           <span className="book-author">{book.author}</span>
           <div className="btn-container">
             <button type="button" className="btn">Comment</button>
             <div className="line" />
-            <button type="button" className="btn" onClick={() => removeBook(book.item_id)}>Remove</button>
+            <button type="button" className="btn" onClick={() => removeBook(book.itemId)}>Remove</button>
             <div className="line" />
             <button type="button" className="btn">Edit</button>
+
+          </div>
+        </div>
+        <div className="progress-bar">
+          <div className="progress-container">
+            <div style={{ width: 70, height: 70 }}>
+              <CircularProgressbar value={1} />
+            </div>
           </div>
+          <div className="text">
+            <span className="Percent-Complete">2%</span>
+            <span className="Percent-Complete1">Completed</span>
+          </div>
+        </div>
+        <div className="Chapter-bar">
+          <span className="Current-Chapter">Current Chapter:</span>
+          <span className="Current-Lesson">Chapter 17</span>
+          <div className="Rectangle"><span className="update-progress">Update progress</span></div>
         </div>
```

Comment:
- [ ] 👏 Great job adding the placeholder elements! However, there are some styling discrepancies in your implementation. As per the design, the "Current-Chapter" and "Current-Lesson" elements should be on separate lines, but currently, they are not following the design. Additionally, the button does not match the design. I recommend aligning your styling more closely with the design to improve the visual consistency of this section. Keep up the good work! 👍🎨📝

| Your | Design |
| --- | --- |
|  ![image](https://github.com/Obimbo07/book-library/assets/55840999/78760491-0d4b-4cef-a92f-bc68af5e2954)| ![image](https://github.com/Obimbo07/book-library/assets/55840999/42098d44-c7b2-4a7c-a9e7-9ab70f84fb0f)|


- [ ] Additionally, there should be a vertical line between the "progress-bar" and "Chapter-bar," which is currently missing in your code. Please consider following the design and updating your code to include this line for a more accurate representation of the design.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/Obimbo07/book-library/assets/55840999/c86098a4-88c9-49d6-89c1-89020e20943d)| ![image](https://github.com/Obimbo07/book-library/assets/55840999/c84a67f6-6947-43f1-b15e-0b24c80e9b6f)|

Reply:
Done implementing the changes.